src/Sulu/Bundle/WebsiteBundle/Resources/public/js/services/reference-store.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define(function() {
+
+    'use strict';
+
+    var synonyms = {},
+        references = {},
+
+        resolveAlias = function(alias) {
```

Comment:
Now when I am seeing this, I have to say that I totally don't like it... Having a name right is one of the most natural things of programming, and I think this might give us some unpredictable troubles... I would prefer to fail and show the available options instead.